lib/clients/client.ex

```diff
@@ -0,0 +1,40 @@
+defmodule GlassFactoryApi.Clients.Client do
+  @moduledoc """
+  Defines the Client of organization.
```

Comment:
Just an english 🎨: I didn't get that description on the first glance. I then realised that `organization` should have an indefinite article preceding it 🙂 

```suggestion
  Defines the Client of an organization.
```